src/Sulu/Bundle/SecurityBundle/Resources/config/services.xml

```diff
@@ -286,6 +290,13 @@
             <tag name="sulu.context" context="admin"/>
         </service>
 
+        <service id="sulu_security.system_listener" class="Sulu\Bundle\SecurityBundle\EventListener\SystemListener">
+            <argument type="service" id="sulu_security.system_store" />
+            <argument type="service" id="sulu_core.webspace.request_analyzer"/>
+            <argument>%sulu.context%</argument>
+            <tag name="kernel.event_listener" event="kernel.request" />
```

Comment:
I would register it with a higher priority so its available as soon as possible. The webspace is set on `32` maybe go with `30`:

```
Registered Listeners for "kernel.request" Event
===============================================

 ------- ---------------------------------------------------------------------------------------------- ----------
  Order   Callable                                                                                       Priority
 ------- ---------------------------------------------------------------------------------------------- ----------
  #1      Symfony\Component\HttpKernel\EventListener\DebugHandlersListener::configure()                  2048
  #2      Symfony\Component\HttpKernel\EventListener\ValidateRequestListener::onKernelRequest()          256
  #3      FOS\RestBundle\EventListener\ZoneMatcherListener::onKernelRequest()                            248
  #4      Symfony\Component\HttpKernel\EventListener\SessionListener::onKernelRequest()                  128
  #5      Symfony\Component\HttpKernel\EventListener\LocaleListener::setDefaultLocale()                  100
  #6      Symfony\Component\HttpKernel\EventListener\FragmentListener::onKernelRequest()                 48
  #7      Sulu\Bundle\WebsiteBundle\EventListener\RouterListener::onKernelRequest()                      32
  #8      Sulu\Bundle\WebsiteBundle\Routing\RequestListener::onRequest()                                 31
  #9      Symfony\Component\HttpKernel\EventListener\LocaleListener::onKernelRequest()                   16
  #10     Symfony\Component\HttpKernel\EventListener\LocaleAwareListener::onKernelRequest()              15
  #11     Sulu\Bundle\WebsiteBundle\EventListener\TranslatorListener::onKernelRequest()                  14
  #12     FOS\RestBundle\EventListener\BodyListener::onKernelRequest()                                   10
  #13     Symfony\Bundle\SecurityBundle\Debug\TraceableFirewallListener::configureLogoutUrlGenerator()   8
  #14     Symfony\Bundle\SecurityBundle\Debug\TraceableFirewallListener::onKernelRequest()               8
  #15     Sulu\Bundle\SecurityBundle\EventListener\UserLocaleListener::copyUserLocaleToRequest()         0
  #16     Sulu\Bundle\SecurityBundle\EventListener\SystemListener::onKernelRequest()                     0
 ------- ---------------------------------------------------------------------------------------------- ----------
```

Reply:
Good catch! Going with `24`, to keep binary methodology and leave some room between the listeners.

Reply:
`24` sounds good for me.